packages/selinux-policy/Makefile

```diff
@@ -108,9 +108,9 @@ ifneq ($(CONFIG_BUILD_ENFORCING_MODE),n)
 else
 	ENFORCING_MODE := permissive
 endif
-#ifneq ($(CONFIG_BUILD_UNCONFINED_TOOR),n)
-#	ENABLE_MODULES += toor
-#endif
+ifneq ($(CONFIG_BUILD_UNCONFINED_TOOR),n)
+	SEPARATE_PKGS += toor
```

Comment:
I think the toor module should always be built as a separate package

Reply:
Do you mean it should always be present, but not inserted into the store?

Or do you mean, CONFIG_BUILD_UNCONFINED_TOOR is "y", it should be put in a separate RPM?

If it is the second one, SEPARATE_PKGS should override whatever is in modules.conf, and then place the PP in a separate RPM.

The paptch dropped the commented out piece that only enabled it, and didn't place it in a separate RPM.

Reply:
It should always be built and included in its own package (IOW, always present in SEPARATE_PKGS).  The kickstart should decide if it gets installed or not.

Reply:
That makes sense that removes the logic in the policy RPM Makefile and leaves it in the ks. I'll update this.